msbuild/Xamarin.Mac.Tasks/Xamarin.Mac.ObjCBinding.CSharp.targets

```diff
@@ -67,15 +67,15 @@ Copyright (C) 2014 Xamarin Inc. All rights reserved.
 			AllowUnsafeBlocks="$(AllowUnsafeBlocks)"
 			ApiDefinitions="@(ObjcBindingApiDefinition)"
 			BaseLibDll="$(BaseLibDllPath)"
-			CompilerPath="$(CscPath)"
+			CompilerPath="/Library/Frameworks/Mono.framework/Commands/csc"
```

Comment:
I know this is very unlikely to change but do we really want a hard path here?

Reply:
I believe the Commands are considered "safe" paths to hard code an similar to a public API. I could shove it in a variable if desired, just seemed silly for one usage.

CscPath was empty in my testing.

Reply:
I'm definitively not an expert but I would expect that a well known `$(CscPath)` is better because it can be overridden before ?
By _empty_ do you mean it did not work/find `csc` ? if so then it might need an empty check and default to your value which **is** a safe default - but maybe not what you want to use, e.g. when testing a different compiler like we'll likely do for default method interfaces ;-)